### DISCUSSION TITLE
[MTP / Portable Devices]: add devices storages to Drives List

### DISCUSSION THREAD

**Discussion by PetbkA** [doublecmd/doublecmd#1211] [Suggestions]
> Double Commander
> Version: **1.1.0 beta**
> Revision: 32
> Commit: **cd3a69c65**
> Build date: 2023/08/05
> Lazarus: 2.2.7.0
> FPC: 3.2.2
> Platform: **x86_64-Win64-win32/win64**
> OS version: **Windows 10 Home 22H2 x86_64**

To access e.g. Android phone's storages, I have to go to _VFS List → My Computer → Phone_, and only then I'll see its storages.

I suggest to add portable device'storages to [_Drives List_](https://github.com/doublecmd/doublecmd/assets/22843816/7fd96e5f-a9f6-4c69-9d5c-0f95eec85c02) if they become available

**Comment by alexx2000** [comment]
I plan to add a whole device to drive list like Explorer/Total Commander does.

**Comment by ags1234** [comment]
That would be great to have, thanks 👍
